### PR TITLE
Add spanish locale

### PR DESCRIPTION
**Pull request by brielov**

### CODE REVIEW

**Comment by diegoulloao** [comment]
@LoicMahieu what about this? I need es-ES !!

File: src/locales/es.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+export const boolean: LocaleObject['boolean'] = {};
+
+export const object: LocaleObject['object'] = {
+  noUnknown:
+    '${path} campo no puede tener llaves no especificados en la forma del objeto',
+};
+
+export const array: LocaleObject['array'] = {
+  min: '${path} campo debe tener al menos ${min} elementos',
+  max: '${path} campo debe tener menos de o igual a ${man} elementos',
```
Comment (diegoulloao) [inline src/locales/es.ts:67]: here is `${max}` instead of `${man}`

**Comment by LoicMahieu** [comment]
Duplicate, https://github.com/LoicMahieu/yup-locales/pull/12

Thanks.

**Comment by LoicMahieu** [comment]
+ yup-locales@1.1.0

**Pull request closed without merging**